shared/src/business/entities/EntityConstants.js

```diff
@@ -479,6 +479,11 @@ const SYSTEM_GENERATED_DOCUMENT_TYPES = {
     documentType: 'Notice of Docket Change',
     eventCode: 'NODC',
   },
+  noticeOfChangeToRemoteProceeding: {
```

Comment:
Does this needed to be added to `courtIssuedEventCodes.json` or any of the others?

Reply:
@rachelschneiderman it seems like other auto-generated trial related notices were also defined in `EntityConstants.js` without a counterpart in any of the document related JSONs (maybe to avoid them showing up in certain dropdowns?), so we decided to follow that pattern!